app/src/endpoints.c

```diff
@@ -59,6 +63,9 @@ bool zmk_endpoint_instance_eq(struct zmk_endpoint_instance a, struct zmk_endpoin
 
     case ZMK_TRANSPORT_BLE:
         return a.ble.profile_index == b.ble.profile_index;
+
+    case ZMK_TRANSPORT_NONE:
```

Comment:
Super nitpick: not really a big deal, but it would be nice for all of the switches to have the cases in the same order as the enum values are defined.